conf/test.config

```
@@ -5,25 +5,33 @@
     Defines input files and everything required to run a fast and simple pipeline test.
 
     Use as follows:
-        nextflow run nf-core/pacvar -profile test,<docker/singularity> --outdir <OUTDIR>
+        nextflow run nf-core/pacvario -profile test,<docker/singularity> --outdir <OUTDIR>
 
 ----------------------------------------------------------------------------------------
 */
 
+process {
+    resourceLimits = [
+        cpus: 2,
+        memory: '15.GB',
+        time: '6.h'
+    ]
+}
+
 params {
     config_profile_name        = 'Test profile'
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
-    // Limit resources so that this can run on GitHub Actions
-    max_cpus   = 2
-    max_memory = '6.GB'
-    max_time   = '6.h'
+    input  = params.pipelines_testdata_base_path + 'refs/heads/pacvar/pacbio_data/samplesheet_longread_puretarget.csv'
```

Perhaps you could change the base of `pipelines_testdata_base_path` so that you can remove the first part of the path here and below? 
```suggestion
    input  = params.pipelines_testdata_base_path + 'pacbio_data/samplesheet_longread_puretarget.csv'
```

---

Linting fails if I change the testdata_base_path which is standardized across pipelines

---

I think you should be able to change it, see https://github.com/nf-core/methylseq/blob/d16f8f81e812d86f4d297e69de3f001628dfd5b4/nextflow.config#L102 for example.

---

maybe it is because I included refs/heads - will try with just pacvar